### DISCUSSION TITLE
Libre office

### DISCUSSION THREAD

**Discussion by shadow-absorber** [catppuccin/catppuccin#2350] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

dracula has a theme for libre office here: [https://github.com/dracula/libreoffice](https://github.com/dracula/libreoffice)

### Any additional comments?

N/A

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by EuroNutellaMan** [comment]
I'd love this too

**Comment by romanskorobagatko** [comment]
Would be really nice to have.

**Comment by NeonGamerBot-QK** [comment]
WIP. https://github.com/NeonGamerBot-QK/catppuccin-libre-office

**Comment by NeonGamerBot-QK** [comment]
https://github.com/catppuccin/catppuccin/issues/2527

**Comment by sgoudham** [comment]
Closing as further discussion should take place in https://github.com/catppuccin/catppuccin/issues/2527
